hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricherTest.java

```diff
@@ -25,6 +16,17 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hypertrace.traceenricher.enrichment.enrichers.SpanTypeAttributeEnricher.CLIENT_KEY;
+import static org.hypertrace.traceenricher.enrichment.enrichers.SpanTypeAttributeEnricher.CLIENT_VALUE;
+import static org.hypertrace.traceenricher.enrichment.enrichers.SpanTypeAttributeEnricher.SERVER_VALUE;
+import static org.hypertrace.traceenricher.enrichment.enrichers.SpanTypeAttributeEnricher.SPAN_KIND_KEY;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 public class SpanTypeAttributeEnricherTest extends AbstractAttributeEnricherTest {
```

Comment:
I am curious about this test. In https://github.com/hypertrace/hypertrace-trace-enricher/pull/36/files#diff-9cc8b5577cab457f08f2a2e0ec514d26R181 we added quite some `if`s but the test class hasn't changed too much, maybe a table test would do the trick?

Reply:
The change is a very small one. But for some weird reason the function location has changed. I'll move the function to old location and that should generate the correct diff